sidecar/internal/csiaddonsnode/csiaddonsnode.go

```diff
@@ -173,7 +178,7 @@ func (mgr *Manager) getCSIAddonsNode() (*csiaddonsv1alpha1.CSIAddonsNode, error)
 		},
 		Spec: csiaddonsv1alpha1.CSIAddonsNodeSpec{
 			Driver: csiaddonsv1alpha1.CSIAddonsNodeDriver{
-				Name:     mgr.Driver,
+				Name:     driver,
```

Comment:
Question: Why not set it `mgr.Driver = driver`?

Reply:
because `mgr.Driver` was dropped from the struct :)